frontend/src/static/js/utils/app-router.ts

```diff
@@ -55,6 +55,11 @@ export const navigateToUrl = (url: string, event?: MouseEvent) => {
     return;
   }
 
+  const oldPath = window.location.pathname + window.location.search;
+  if (oldPath === url) {
```

Comment:
Could you try this and report back?
```suggestion
  const oldUrl = window.location.href;
  if (oldUrl === url) {
```

Reply:
@jcscottiii I have made the change, and from overview page sending the complete URL instead of path